src/convertToObject.js

```diff
@@ -7,6 +7,20 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .trim()
+    .split(';')
+    .map((style) => style.trim())
+    .filter((style) => style)
+    .reduce((stylesObject, style) => {
+      const [property, value] = style.split(':').map((part) => part.trim());
```

Comment:
The destructuring assignment within the `reduce` callback is done correctly. It's good to see that you're trimming both the property and the value to ensure no extra whitespace is included in the keys and values of the resulting object.